### ISSUE TITLE
The application “monolith-code.app” can’t be opened.

### ISSUE THREAD

**Issue by SeaMac** [Haeri/MonolithCode2#18]
Mac OS 10.13.6 .... download app ... and open it from the downloads folder.

"The application “monolith-code.app” can’t be opened." is all that is displayed.

Problem reproducible.

The web site at https://haeri.github.io/MonolithCode2/ doesn't say what system requirements it needs. 

In fact that web page is really incomplete ... no features list or description or anything about this app.  Just a sketchy looking Download button to an APP that doesn't work.

**Comment by Haeri** [comment]
Hi @SeaMac 

Thanks for pointing out the issue. I'm looking into it right now. 

The idea of the project is to have a super minimalistic design. This idea also carried over to the website. I'm trying to strip away everything that is not absolutely necessary. However, the editor has some cool features that I do indeed would like to communicate. I just haven't figured out a nice way of doing presenting the feature list. 

But here are some highlights for now
- Syntax highlighting for: batchfile, c, cpp, csharp, css, dart, dockerfile, go, html, java, javascript, json, latex, makefile, markdown, php, plaintext, powershell, python, sql, typescript, xml, yaml
- Customizable build and run functionality for most of the compiled languages
- Built-in console
- Realtime Markdown preview with MathJax symbols
- Markdown to PDF exporter
- HTML preview
- Inline math calculator
- Beautifier

But you are right with the "sketchy" looking download button. I tried to change the colors a bit to make it fit better. 👍

**Comment by Haeri** [comment]
The good news is that I found out what the issue was:
When uploading the artifacts, the files are archived as a zip. Unfortunately, the zip format does not retain permissions, so the executable loses its `chmod x`. So the fix is simple, you would have to just add back chmod in the terminal:
```shell
chmod +x monolith-code.app/Contents/MacOS/monolith-code
```

The bad news is that the macOS version seems to require some more love as I found some more issues down the road with GPU errors and rendering problems. Unfortunately, I don't own a mac and so it is difficult for me to debug these errors right now. 

I'll remove the macOS download button in the meantime until I find a solution.
